build.sbt

```diff
@@ -189,7 +189,7 @@ lazy val midas      = ProjectRef(firesimDir, "midas")
 lazy val firesimLib = ProjectRef(firesimDir, "firesimLib")
 
 lazy val firechip = (project in file("generators/firechip"))
```

Comment:
Also while we are at it, could we change the `icenet` project and the `firechip` project to use `conditionalDependsOn` (and delete the `testchipip` dependency). This either pulls `testchipip` from maven or it just adds the dependency from the repo.

Reply:
Why are we even pulling that conditionally from maven? These changes seem out of scope to the PR.

Reply:
Paul Rigge wants to support a case where you might use `testchipip` from maven. We can do this in a separate PR. Im going to go ahead and approve this PR.